mount/lookup_unix.go

```diff
@@ -29,6 +29,11 @@ import (
 func Lookup(dir string) (Info, error) {
 	dir = filepath.Clean(dir)
 
+	dir, err := filepath.EvalSymlinks(dir)
+	if err != nil {
+		return Info{}, fmt.Errorf("failed to resolve symlink for %q: %w", dir, err)
```

Comment:
If the above fails dir will (haven't checked all failure paths but seems like most, and it's just a go convention) be an empty string, so including it in the error msg won't be too helpful.

```suggestion
	resolvedDir, err := filepath.EvalSymlinks(dir)
	if err != nil {
		return Info{}, fmt.Errorf("failed to resolve symlink for %q: %w", dir, err)
```

something like the above, whatever you want to use for the new var name is up to you.

Reply:
thank you sir! updated